src/Cedar/UdpAccel.c

```diff
@@ -63,33 +63,30 @@ void UdpAccelPoll(UDP_ACCEL *a)
 			if (a->UseUdpIpQuery && a->UdpIpQueryPacketSize >= 8 && CmpIpAddr(&a->UdpIpQueryHost, &src_ip) == 0 &&
 				src_port == a->UdpIpQueryPort)
 			{
+				/*
 				// Receive a response of the query for IP and port number
 				IP my_ip = {0};
 				UINT myport = 0;
 				BUF *b = MemToBuf(a->UdpIpQueryPacketData, a->UdpIpQueryPacketSize);
-
-
 				FreeBuf(b);
+				*/

```

Comment:
is this for testing? Otherwise, I’d remove dead code.

Reply:
I agree, as far as I know all commented code in the file is for VPN Gate.